python/pyspark/storagelevel.py

```diff
@@ -19,6 +19,14 @@
 
 
 class StorageLevel(object):
+    DISK_ONLY: 'StorageLevel'
```

Comment:
This is already covered with stubs and I think it is better to avoid annotations in `.py` unless they're strictly needed.